docs/meetings/2022/TechArea20221122.md

```diff
@@ -27,27 +27,45 @@
 | 6            | +3      | Ready for Testing |
 | 0            | +0      | Ready for Release |
 
+-  Brian will be tossing more things into backlog to make sense of prios;
+   in need of cleaning
+
 ## OSG Software Team
 
 -  AI (Mat): Assist Mats and HTCondor devs in getting container universe working on the OSPool
--  AI (Mat, Carl) EL 9 groundwork
+-  AI (Mat): scitokens ticket for Steve Timm
+-  AI (Mat, Carl) EL 9 groundwork:
+   -  Carl: for next week, add VMU images for EL9 (SOFTWARE-5337)
+   -  After that: OSDF tickets selected for development: SOFTWARE-5296, SOFTWARE-5295
 
 ### Discussion
 
-None this week
+- Marco: glideinwms: frontend is not reporting properly to monitoring
 
 ### Support Update
 
+-   BrianL: discusses Open Science Pool token registry; will manually create a token for comanage after discussion with
+    Brians B,A
 -   University of Arkansas Little Rock (BrianL): working around an issue with ORCID and the self-service OS Registry
 -   NERSC (Mat, BrianB, BrianL): got a patched XRootD into testing for Lisa to install to test issues with missing
     Role/Capability in her VOMS certificate. That seems to be fixed but there are still ongoing issues.
+    Mat: not getting logging we are supposed to for XRootD server at NERSC.
+-   CalTech startd cron not getting parsed correctly; BrianL will talk to Todd about it
+-   Carl: DESY: don't want other VOs they host
+-   Carl: unregistered gratia probes: we want to close for moral vicotry,
```

Comment:
Lol, that's right.  I don't remember why but for some reason i left that in there on purpose.  Probably because it sounded amusing.  Good catch anyway though.